pkg/registry/pod/etcd/etcd.go

```diff
@@ -218,6 +220,38 @@ type LogREST struct {
 	kubeletConn client.ConnectionInfoGetter
 }
 
+// Max length read from the body of a log location which returns error status
+const (
+	maxReadLength = 1000
```

Comment:
I think our api status object can be >> 1000.  Would go to 50k which is trivial amount of memory.